modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -343,6 +343,11 @@ up    <string name="player_sleep_in_one_chapter">Sleeping in 1 chapter</string>
     <string name="player_up_next_move_to_top">Move to top</string>
     <string name="player_up_next_select">Select</string>
     <string name="player_up_next_time_remaining">Total time remaining %s</string>
+    <string name="player_up_next_time_left">%s left</string>
+    <plurals name="player_up_next_header_title">
+        <item quantity="one" formatted="true">%1$d episode - %2$s left</item>
+        <item quantity="other" formatted="true">%1$d episodes - %2$s left</item>
```

Comment:
We have these separately `episodes_singular`, `episodes_plural`, and `time_left `. Would it be better to use those rather than creating a new string to combine them?

Reply:
Regular strings divided into `singular` and `plural` do not handle plurality rules correctly. English has only two forms `one` and `zero`. But other languages might have different forms. For example in Polish you'd say.

```
0 episodes - 0 epizodów
1 episode - 1 epizod
2/3/4 episodes - 2/3/4 epizody
5 episodes - 5 episodów
```

Reply:
Oh nice! Thanks for explaining @MiSikora! I'll leave it as is then.

Reply:
Changing `episodes_singular` and `episodes_plural` to use Android plurals would good. I more meant that it might be a good idea to not combine episode and left in the same string.

Reply:
> I more meant that it might be a good idea to not combine episode and left in the same string.

But the position of these might depend entirely on language. In some languages "time left" might make more sense to be displayed at the beginning.